lib/azure/armrest/storage/disk_service.rb

```diff
@@ -11,6 +11,115 @@ class DiskService < ResourceGroupBasedService
         def initialize(configuration, options = {})
           super(configuration, 'disks', 'Microsoft.Compute', options)
         end
+
+        # Get the raw blob information for a managed disk. This is similar to
```

Comment:
@djberg96 I always like your documentation with examples. Very informative and practical. 👏